src/web/rustdoc.rs

```diff
@@ -448,8 +452,10 @@ mod test {
             let web = env.frontend();
             assert_success("/", web)?;
             assert_success("/crate/buggy/0.1.0/", web)?;
+            assert_success("/crate/buggy/0.1.0/", web)?;
```

Comment:
I think this was a typo.

```suggestion
            assert_success("/crate/buggy/0.1.0", web)?;
```

Reply:
It was a typo